libs/category/src/reducers/index.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { TestBed } from "@angular/core/testing";
+import { StoreModule, combineReducers, Store, select } from "@ngrx/store";
+
+import { DaffCategoryFactory } from '@daffodil/category/testing';
+
+import { DaffCategoryLoadSuccess } from "../actions/category.actions";
+import * as fromCategory from './index';
+import { DaffCategory } from "../models/category";
+
+describe('selectCategoryState', () => {
+
+  let store: Store<fromCategory.State>;
+  const categoryFactory: DaffCategoryFactory = new DaffCategoryFactory();
+  let mockCategory: DaffCategory;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        StoreModule.forRoot({
+          category: combineReducers(fromCategory.reducers),
+        })
+      ]
+    });
+
+    mockCategory = categoryFactory.create();
+    store = TestBed.get(Store);
+
+    store.dispatch(new DaffCategoryLoadSuccess(mockCategory));
+  });
+
+  describe('selectCategoryState', () => {
+
+    describe('selectCategoryLoading', () => {
+
+      it('selects the loading state of the category', () => {
+        store.pipe(select(fromCategory.selectCategoryLoading)).subscribe((categoryLoadingState) => {
```

Comment:
Could these be replaced with `jasmine-marble` tests? I think these are never unsubscribed and create a memory leak.

Reply:
done